### ISSUE TITLE
The loss has always been around 0.7

### ISSUE THREAD

**Issue by liuzuo-byte** [lucidrains/byol-pytorch#85]
![image](https://user-images.githubusercontent.com/84719243/200246931-eabf1ff4-da30-46ad-8bfc-9544fdf5bff0.png)

**Comment by liuzuo-byte** [comment]
it train on my data
